h2o-algos/src/main/java/hex/tree/SharedTreeModelGBMDRF.java

```diff
@@ -0,0 +1,159 @@
+package hex.tree;
+
+import hex.genmodel.algos.tree.SharedTreeNode;
+import hex.genmodel.algos.tree.SharedTreeSubgraph;
+import hex.genmodel.algos.tree.*;
+import water.*;
+import water.fvec.Chunk;
+import water.fvec.Frame;
+import water.fvec.NewChunk;
+import water.fvec.Vec;
+import water.util.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import hex.Model;
+import water.Key;
+
+public abstract class SharedTreeModelGBMDRF<
```

Comment:
why not SharedTreeModelWithContributions? too long?

Reply:
I agree. Done.